### PR TITLE
Create pt.yml

### PR DESCRIPTION
**Pull request by mayconmk**
dear Alina;
According to your message, follow here the second pull request of the full translate for Brasilian Portuguese translation file.

version beta 2 (with some error fixed)

(I'll verify again the pull-request errors)

BR;
Maycon

### CODE REVIEW

**Comment by nag5000** [comment]
Amazing work, thank you!

Please do not duplicate PRs for each patch (#9, #10, #11).

File: translations/pt.yml [hunk 1]
```diff
@@ -0,0 +1,3280 @@
...
+  A avaliação está marcada para exclusão!<br> Ela será excluída depois que você pressionar
+  "Salvar". Clique aqui para desfazer.
+instruction_urgent:
+  btn: Concluir agora
+  btn_expl: A pontuação será dada com base nas avaliações recebidas no momento.
+  fail: "Falha ao concluir a avaliação, tente novamente mais tarde"
+  finished_expl: >-
+    {count, plural, =1 {Resultado da avaliação: {percentage}%. Por iniciativa do instrutor, a
+    pontuação foi baseada em uma avaliação única {date}.} other {Resultado da avaliação:
+    {percentage}%. Por iniciativa do instrutor, a pontuação foi baseada em # avaliações {date}.}
```
Comment (nag5000) [inline translations/pt.yml:1169]: Fix of `Expected "=", "\\#", "\\\\", "\\u", "\\{", "\\}", "}" or [^{}\\\0-\x1F\x7f \t\n\r] but end of input found.`:
```suggestion
    {percentage}%. Por iniciativa do instrutor, a pontuação foi baseada em # avaliações {date}.}}
```

**Comment by mayconmk** [comment]
Dear Aleksey Nagovitsyn;

     Sorry for mistakes that happened during the Pull Requests... I'm new using the GitHub.
     I already commited your review suggestion. And if the pt.yml file don't have more technical errors, you can consider it as the fisrt final version for Brazilian Portuguese (pt-BR) translation.

    What are the next step now?

Best Regards;
Maycon

**Comment by nag5000** [comment]
Errors `"xxx" was not found in "en", "ru"` means `"xxx"` translation is outdated. You can just delete them. But I can do it when I have time.

I will need to do some additional technical checks, and then we can accept the translation.
After merging this PR new translation will be live on https://stepik.org within ~2 weeks. It'll probably be June 18th.

**Comment by nag5000** [comment]
Date of the next release has changed: June 18 -> 22/23.

**Comment by nag5000** [comment]
@mayconmk I'd like to clarify the locale name. 
As you said, it's a Brasilian Portuguese translation. In this case, the file should be called `pt-br.yml` instead of `pt.yml`. And the native name of the locale will be `Português Brasileiro`, not `Português` (it will be used in our language selector). Is this correct?

**Comment by mayconmk** [comment]
@nag5000 most of cases the websites here in Brazil, when selecting language is available, we find the option "Português" (only one word). You don't need to put the word "Brasileiro". Only "Português" on language selector of Stepik website, is enough.

**Comment by mayconmk** [comment]
@nag5000 talking about translation file be called pt-br.yml no problem. You can change it.

**Comment by mayconmk** [comment]
Dear @nag5000 I'm starting now to review the portuguese version, but I have follow question: What about the Stepik mobile App? I found some students submitting their activities throw mobile version of Stepik plattform. The fact is that on mobile version there are only two language options: English and Russian... Is it possible we think about translating the app language too? I can help you again on this task.

**Comment by mayconmk** [comment]
Dear @nag5000 I'm still waiting your reply about my question... Did you receive it? @

**Comment by al3xnag** [comment]
@mayconmk I apologize for my silence. Stepik mobile team has no plans to support community languages yet.

**Pull request merged**